test/fixtures/test-runner/snapshots/unit.js

```diff
@@ -22,3 +22,7 @@ test('`${foo}`', async (t) => {
   const options = { serializers: [() => { return '***'; }]};
   t.assert.snapshot('snapshotted string', options);
 });
+
+test('escapes', async (t) => {
```

Comment:
Can you add some more escape characters that are also used by `templateEscape()`.